apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -1446,4 +1447,30 @@ describe('LIVECHAT - rooms', function () {
 			expect(response.body.filters.find((f: IOmnichannelRoom['source']) => f.type === 'api')).to.not.be.undefined;
 		});
 	});
+
+	describe('it should mark room as unread when a new message is sent', () => {
+		let room: IOmnichannelRoom;
+		let visitor: ILivechatVisitor;
+		let totalMessagesSent = 0;
+		let departmentWithAgent: Awaited<ReturnType<typeof createDepartmentWithAnOnlineAgent>>;
+
+		before(async () => {
```

Comment:
no big deal, but instead of using `before` I feel this should be part of the test it self =)

I mean, you should use `before` to prepare something to be used in the next tests cases (`it`), but this before looks *very specific*, that's why.

or maybe only the `updateSetting` should be in the `before` statement.. or even you can create an `after` statement to clean everything up (like removing the department, room, etc)

Reply:
Makes sense. Moved that bunch of operations to a separate `it` hook